### DISCUSSION TITLE
WIFI-Configuration

### DISCUSSION THREAD

**Discussion by Pedritoprint** [dl9rdz/rdz_ttgo_sonde#457] [TTGO firmware]
Is it possible to login to a known Network (SSID) with Username an Password?
I'm on a campground in Spain and we need to connect to the WiFi the SSID, Username and Password.

**Comment by dl9rdz** [comment]
Unfortunately for technical reasons it is not possible to connect to a WiFi that requires a captive portal for authentication.
This would basically require a full web browser on the client device... Not suitable for the ESP32

**Reply by Pedritoprint** [reply]
Thanks for your quick answer. Greetings, HB9CMI/Peter
